cli/api/client/app/http_client.go

```diff
@@ -0,0 +1,38 @@
+package app
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+	"os"
+)
+
+func HttpClient(method string, endpoint string, request interface{}, response interface{}) error {
+	uri := os.Getenv("API_URI")
+	req, _ := http.NewRequest(method, uri+endpoint, nil)
+
+	if request != nil {
+		req.Header.Set("Content-Type", "application/json")
+		reqBody, err := json.Marshal(request)
+		if err != nil {
+			return err
+		}
+		req.Body = io.NopCloser(bytes.NewBuffer(reqBody))
+	}
+
+	client := http.Client{}
```

Comment:
一応タイムアウトも設定しておいた方がいいかも

```suggestion
	client := http.Client{Timeout: 30 * time.Second}
```